tunneling/fastify/index.js

```diff
@@ -0,0 +1,49 @@
+const Fastify = require("fastify");
+
+const fastify = Fastify({ logger: true, bodyLimit: 100 * 1048576 }); //TODO: adjust bodyLimit to suit your needs (Default value is 100mb to match cloudflare free plan limits)

```

Comment:
```suggestion
// Allow body up to 100 MB
const fastify = Fastify({ bodyLimit: 100 * 1048576 });
```

I think this is fine like this!